include/mbedtls/platform_util.h

```diff
@@ -25,7 +25,18 @@
 #ifndef MBEDTLS_PLATFORM_UTIL_H
 #define MBEDTLS_PLATFORM_UTIL_H
 
+#if !defined(MBEDTLS_CONFIG_FILE)
+#include "mbedtls/config.h"
+#else
+#include MBEDTLS_CONFIG_FILE
+#endif
+
+#include "mbedtls/platform_time.h"
+
 #include <stddef.h>
+#if defined(MBEDTLS_HAVE_TIME_DATE)
```

Comment:
Shouldn't this have !defined(MBEDTLS_PLATFORM_GMTIME_ALT)?

Reply:
@AndrzejKurek No, the prototype for `mbedtls_platform_gmtime_r()` needs the structures from `time.h`.